assets/css/style.css

```diff
@@ -252,3 +229,82 @@ hr {
   border: 0.5px;
   margin-top: 10px;
 }
+
+.chev i {
+  color: #acb7c3;
+}
+
+.contact-me {
+  background: #fff;
+}
+
+.contact {
+  background-image: url("../images/contact-bg.svg");
+  background-position: center 100px;
+  background-size: auto;
+  background-repeat: no-repeat;
+  border-top-left-radius: 160px;
+  background-color: #6070ff;
+}
```

Comment:
- [ ] Hello, great effort working on the spme of the changes requested by the previous reviewer. But as stated by the reviewer, we are implore to stick to the figma design as much as possible. Currently,  your contact section design is missing the background image as we have it in the original figma design. I would kindly suggest you add it and be sure it is showing properly the way we have it in the original design.

Your design :point_down: 

![Screenshot_20230511-152040_Chrome](https://github.com/Izebiafe/my-protfolio/assets/78634841/f1fb62ef-eba2-4dea-b038-6623d1d65e7e)